sanic/mixins/runner.py

```diff
@@ -424,7 +424,7 @@ def _helper(
 
         self.motd(self.serve_location)
 
-        if sys.stdout.isatty() and not self.state.is_debug:
+        if sys.stdout and sys.stdout.isatty() and not self.state.is_debug:
```

Comment:
For some reason GH is not giving me the option to create a suggestion here... :thinking: 

Nevetheless, can you change this whole block like this:

```python
        if not self.state.is_debug:
            message = (
                "Sanic is running in PRODUCTION mode. Consider using "
                "'--debug' or '--dev' while actively developing your "
                "application."
            )
            if sys.stdout and sys.stdout.isatty():
                message = "".join([Colors.YELLOW, message, Colors.END])
            error_logger.warning(message)
```

Reply:
At some point we can abstract some of the color stuff, but for now this at least keeps the message still populating.